hibernate-core/src/main/java/org/hibernate/dialect/OracleServerConfiguration.java

```diff
@@ -45,40 +56,79 @@ public int getDriverMinorVersion() {
 		return driverMinorVersion;
 	}
 
-	public OracleServerConfiguration(boolean autonomous, boolean extended) {
-		this( autonomous, extended, 19, 0 );
+	public OracleServerConfiguration(boolean autonomous, boolean extended, boolean applicationContinuity) {
+		this( autonomous, extended, applicationContinuity, 19, 0 );
 	}
 
 	public OracleServerConfiguration(
 			boolean autonomous,
 			boolean extended,
+			boolean applicationContinuity,
```

Comment:
Please create a new constructor and delegate to that defaulting to `false` for the new parameter.